### ISSUE TITLE
Sort order is case sensitive since V2

### ISSUE THREAD

**Issue by t-karcher** [magnusmanske/listeria_rs#61]
I just noticed that sorting by label in V2 completely messes up the list of alpine huts in

https://it.wikipedia.org/wiki/Utente:Tkarcher/Rifugi_delle_Alpi

Last version before V2:
https://it.wikipedia.org/w/index.php?title=Utente:Tkarcher/Rifugi_delle_Alpi&oldid=115918449

New sort order in V2:
https://it.wikipedia.org/wiki/Utente:Tkarcher/Rifugi_delle_Alpi

All lower case letters are suddenly moved to the end. :-(

Could you please restore the old, case-insensitive logic? Thanks!

**Comment by tacsipacsi** [comment]
Non-ASCII characters like Ö are also out of order. See for example https://hu.wikipedia.org/w/index.php?title=Szerkeszt%C5%91:Ato_01/pr%C3%B3balap/Magyarorsz%C3%A1g_wd_k%C3%A9ptelen_telep%C3%BCl%C3%A9sei&oldid=26605691 –  _Ömböly_ is after _Zsurk_, even though alphabetically it’s well before it. [The documentation](https://doc.rust-lang.org/std/cmp/trait.PartialOrd.html#impl-PartialOrd%3Cstr%3E-for-str) explicitly mentions that `partial_cmp` for strings is _not_ an alphabetical sorting. The PHP version did have proper sorting (using [Collator](https://www.php.net/manual/en/class.collator.php))…

**Comment by tacsipacsi** [comment]
Simply lower-casing the values still doesn’t mean proper sort order, _Ömböly_ is still going to end up after _Zsurk_… Listeria should once again use a real collator, for example from the [`icu::collator` crate](https://docs.rs/icu/latest/icu/collator/).
